routing/ann_validation.go

```diff
@@ -142,5 +144,31 @@ func ValidateChannelUpdateAnn(pubKey *btcec.PublicKey,
 			"update %v", spew.Sdump(a))
 	}
 
+	return validateOptionalFields(capacity, a)
```

Comment:
would recommend checking the optional fields before checking the signature since the latter is more expensive. checking the optional fields allows us to fail faster in the event that they're incorrect

Reply:
nice fixed!